cmake/Modules/FindNVTX.cmake

```diff
@@ -0,0 +1,55 @@
+#.rst:
+# FindNVTX
```

Comment:
This requires CMake 3.17 and higher. I already started investigating if it's possible to build a clean integration switching between normal nvtx with shared library found manually or via CUDAToolkit, and the header-only nvtx3 from CUDA 10.0 on (starting CMake 3.25), but that's even harder, because the include paths differ in structure.